code/modules/supermatter/supermatter.dm

```diff
@@ -409,6 +416,19 @@
 
 	Consume(AM)
 
+/obj/machinery/power/supermatter/shard/Bumped(atom/AM)
+	..()
+	if(istype(AM, /obj/item/supermatter_splinter))
+		if(power_loss_modifier >= 2500)
+			visible_message("<span class = 'sinister'>As \the [AM] fuses to \the [src], \the [src] begins to glow an overworldly shimmer as it begins to pull additional mass from the environment around itself...</span>")
+			new/obj/effect/overlay/gravitywell(loc)
```

Comment:
Shouldn't this by `qdel`'d when you spawn the crystal ?

Reply:
It deletes itself